src/themes/default/pages/Checkout.vue

```diff
@@ -123,11 +123,20 @@ export default {
     isValid () {
       let isValid = true
       for (let child of this.$children) {
-        console.log(child)
+        console.log('Child: ', child)
```

Comment:
Please remove the verbose logging here as it's probably not required beyond the dev. process